scripts/setup-minikube-for-linux.sh

```diff
@@ -17,8 +17,8 @@ mkdir -p $HOME/.kube
 touch $HOME/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
-sudo -E minikube start --vm-driver=none
-sudo -E minikube addons enable ingress
+minikube start --driver=docker
```

Comment:
do we have to update deployment guide and samples also?

Reply:
i'll check. we shouldn't be recommending the any driver but maybe we do

Reply:
We just assume users have minikube in the akka management one: https://doc.akka.io/docs/akka-management/current/kubernetes-deployment/deploying.html#deploying-to-minikube
and we link to the docs from the Lagom one